### ISSUE TITLE
Provide Descriptions of User Roles

### ISSUE THREAD

**Issue by DanielSeehausen** [ospin-web-dev/REACTor#464]
Provide a user accessible description of what the different roles do.

Just spitballing here:
Perhaps a button that opens a modal on the access page (or hover + tooltip that doesn't open a modal)? Anchored right all the way across from the `Add User` button?
![image](https://user-images.githubusercontent.com/17087405/72423753-17575980-3785-11ea-8aa7-03e2d9fe0515.png)

Alternatively, github has a nice interface here we could potentially mimic:
![image](https://user-images.githubusercontent.com/17087405/72423837-41108080-3785-11ea-8449-ea175eec36b5.png)

There may be a react component package to provide a rich dropdown. 
Use your best judgement!

The following questions should either be implicitly, or explicitly, answered via the descriptions:

- Is there a role that supersedes all others?
- Who can take the ownership role away from other users?
- What does the little crown in the access table mean?

To go in after #429 has been merged. 
Might be simpler to do after #465 has been completed, so we don't need to update the descriptions
Addresses part of #462

**Comment by vdiezel** [comment]
I started working on this ticket and realized that we have a couple of inconsistencies in the access rights management, e.g. what an operator can do on the device.

I decided to postpone it until we have resolved this.
